### ISSUE TITLE
reminder appears to be buggy

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#93]
```
What steps will reproduce the problem?
1.set $rcmail_config['debug_level'] = 4;
2./usr/bin/php /srv/www/roundcubemail/plugins/calendar/program/reminders.php
3.

What is the expected output? What do you see instead?
something like 
 done [0,044342994689941 seconds runtime].
but my php says
 PHP Fatal error:  Cannot redeclare class Database in /srv/www/roundcubemail/plugins/calendar/program/backend/database.php on line 456
after correcting this i got
 PHP Fatal error:  Cannot redeclare class Utils in /srv/www/roundcubemail/plugins/calendar/program/utils.php on line 4
and finaly my output looks like
 MDB2 Error: unknown error (-1): _doQuery: [Error message: Could not execute statement]
 [Last executed query: UPDATE events SET notified=0 WHERE recurring>86399]
 [Native message: ERROR:  operator does not exist: text > integer
 ZEILE 1: UPDATE events SET notified=0 WHERE recurring>86399

What version of the product are you using? On what operating system?
trunk-r-3878.zip, SLES11SP1 ,
PHP 5.2.6 with Suhosin-Patch 0.9.6.2 (cli) (built: Feb 12 2010 15:26:33)

Please provide any additional information below.
In reminders.php i update the two includes to include_once and
add tics around the questionmark in line 286.

211: include_once INSTALL_PATH . 
'plugins/calendar/program/backend/database.php';
212: include_once INSTALL_PATH . 'plugins/calendar/program/utils.php';
286:   $query = 'UPDATE ' . get_table_name('events') . ' SET notified=? WHERE 
recurring>\'?\'';

kind regards
```

Original issue reported on code.google.com by `conradsh...@googlemail.com` on 12 Aug 2010 at 11:17

**Comment by GoogleCodeExporter** [comment]
```
Did your fix solve the problems?
```

Original comment by `rol...@liebl.ath.cx` on 12 Aug 2010 at 11:29
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
yes, thank you
```

Original comment by `conradsh...@googlemail.com` on 12 Aug 2010 at 12:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 12 Aug 2010 at 2:21

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3878.1.
```

Original comment by `rol...@liebl.ath.cx` on 14 Aug 2010 at 4:47
- Changed state: **Fixed**
